build/images/flowaggregator/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM antrea/base-ubuntu:2.14.0
+
+USER root
+
+COPY bin/flow-aggregator /usr/local/bin/
```

Comment:
Do we expect the consumer of this dockerfile to compile the flow-aggregator code as a pre-requisite?
Maybe it is better to include the compilation steps for flow-aggregator binary in the dockerfile itself as it is being done for Antrea image.

Reply:
Thanks, added.